packages/react-components/src/components/gif.tsx

```diff
@@ -277,15 +228,20 @@ const Gif = ({
             onContextMenu={(e: SyntheticEvent<HTMLElement, Event>) => onGifRightClick(gif, e)}
         >
             <div style={{ width, height, position: 'relative' }} ref={container}>
-                <img
-                    className={Gif.imgClassName}
-                    src={showGif ? fit : placeholder}
-                    style={{ background }}
-                    width={width}
-                    height={height}
-                    alt={getAltText(gif)}
-                    onLoad={showGif ? onImageLoad : () => {}}
-                />
+                <picture>
+                    <source type="image/webp" srcSet={rendition.webp} />
+                    <img
+                        ref={image}
+                        suppressHydrationWarning
```

Comment:
We render with the image src as the image url in SSR, but then first render on the client, this image src will be the placeholder, since we haven't tested with intersection observer. If the image is on the screen, we'll immediately re-render with an image src as the image url. 

This hasn't caused a flicker, since I believe the first pass with the placeholder isn't committed to the DOM. However it does throw the warning, hence `suppressHydrationWarning`